ui/src/main/webapp/js/movieztalk.js

```diff
@@ -4,7 +4,7 @@ moviezApp.config(['$routeProvider',
   function($routeProvider) {
     $routeProvider.
       when('/home', {
-        templateUrl: 'Home.html',
+        templateUrl: 'Movie.html',
```

Comment:
Add separate when for Movie.html
for eg .when('/Movie', {
-        templateUrl:  'Movie.html'}}.